apps/docs-snippets2/scripts/wrap-snippets.ts

```diff
@@ -12,6 +12,20 @@ const wrapperFnContents = readFileSync(wrapperFnFilepath, 'utf-8');
 export const wrapSnippet = (filepath: string) => {
   const snippetContents = readFileSync(filepath, 'utf8');
 
+  /**
+   * Test environment
+   */
+  let testEnvironments = '';
+
+  // Check if the filepath includes '.node' or '.browser'
+  if (filepath.includes('.node')) {
+    testEnvironments = '/**\n * @group node\n */';
+  } else if (filepath.includes('.browser')) {
+    testEnvironments = '/**\n * @group browser\n */';
+  } else {
+    testEnvironments = '/**\n * @group node\n * @group browser\n */';
+  }
+
```

Comment:
We should start documenting these things in a `docs-snippets2/README.md` file.

Reply:
e19b126ef